integration-tests/tests/src/hooks/import-app-before.ts

```diff
@@ -31,4 +31,14 @@ export function importAppBefore(
       Realm.App.Sync.setLogLevel(this.app, logLevel);
     }
   });
+
+  // Delete our app after we have finished, otherwise the server can slow down
+  // (in the case of flexible sync, with lots of apps with subscriptions created)
+  after(async function (this: Partial<AppContext> & Mocha.Context) {
+    if (!this.app) {
+      throw new Error("No app on context when trying to delete app");
+    }
+
+    await deleteApp(this.app.id);
+  });
```

Comment:
I like the change, but I'm also wondering if this could make it harder to inspect the server-side logs when debugging an issue. At some point we might want to have some sort of "manual mode" where the tests will skip importing and deleting an app and instead simply use an app-id provided when starting the test.

Reply:
Maybe I can add an environment context option to skip deleting the app for now, in case you want to inspect the state of the app after a test run?